src/Form/_Form.scss

```diff
@@ -479,10 +479,18 @@ select.form-control {
 
   &.pgn__form-control-invalid input {
     border-color: $form-feedback-invalid-color;
+
+    &:checked {
+      background-image: escape-svg($custom-checkbox-indicator-icon-valid-checked);
```

Comment:
Should these SCSS variables be flipped? I.e., the `valid` variable here is applied to the `.*-invalid` CSS class and vice versa.

Reply:
Good catch. I'm realizing this needs to be moved out of being applied to all checkbox, switches and radios as well.